.travis.yml

```diff
@@ -4,8 +4,10 @@ sudo: false
 language: node_js
 
 node_js:
-  - "0.10"
   - "4.0"
+  - "4.1"
```

Comment:
confirm me you don't want to test against 4.2.2. LTS, and latest 5.1?
I'll only do 4.1 otherwise